analyses/README.md

```diff
@@ -1,77 +1,77 @@
+| [molecular-subtyping-CRANIO](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/master/analyses/molecular-subtyping-CRANIO)                                      | `histologies-base.tsv`   `snv-consensus-plus-hotspots.maf.tsv.gz`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Molecular subtyping of craniopharyngiomas samples [#810](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/810)                                                                                                                                                               | No                                                                                                                                                                                                            | `results/CRANIO_molecular_subtype.tsv`                                                                                                                                                                                                                                                                                                                                                                                     
```

Comment:
Same here. I think we should change it to histologies-base

Reply:
Here what I mean is for `molecular-subtyping-MB` module.

Reply:
done